frame/support/src/storage/mod.rs

```diff
@@ -802,6 +802,18 @@ pub struct PrefixIterator<T, OnRemoval = ()> {
 	phantom: core::marker::PhantomData<OnRemoval>,
 }
 
+impl<T, OnRemoval1> PrefixIterator<T, OnRemoval1> {
+	pub fn into_other<OnRemoval2>(self) -> PrefixIterator<T, OnRemoval2> {
```

Comment:
`pub` item needs documentation.

Reply:
And I would call this `convert_on_removal`. `into_other` isn't really expressing.